enterprise/vmsupport/kernel/microvm-kernel-x86_64-v5.15.config

```diff
@@ -1,27 +1,28 @@
+# Config copied from https://github.com/firecracker-microvm/firecracker/blob/main/resources/guest_configs/microvm-kernel-ci-x86_64-6.1.config
+# BuildBuddy-specific modifications:
+# - Set CONFIG_PCI=y (see https://github.com/firecracker-microvm/firecracker/issues/4881)
+
 #
```

Comment:
Confused by the diff here. The file name mentions v5.15 but the code references v6.1?

Reply:
Woops, I messed something up here. Will fix

Reply:
Done